scripts/rspec/rspec.ps1

```diff
@@ -267,7 +210,7 @@ pushd $SonarpediaFolder
 
 if ($RuleKey) {
     if ($RspecBranch) {
-        java "-Dline.separator=`n" -jar $RuleApiJar generate -rule $RuleKey -branch $RspecBranch
+#FIXME        java "-Dline.separator=`n" -jar $RuleApiJar generate -rule $RuleKey -branch $RspecBranch
```

Comment:
Will you fix it before merging it or you plan to fix it in a follow up PR?

Reply:
Good catch, this should not be there. And that's why I use `FIXME` that should not make it to master in cases like this 😆